src/file-upload/dropzone.tsx

```diff
@@ -89,7 +90,7 @@ const Component = (
                 accept={accept}
                 capture={capture}
                 multiple={multiple}
-                data-testid="dropzone-input"
+                data-testid={`${testId}-input` || "dropzone-input"}
```

Comment:
This causes a breakage in the test as it is looking for `dropzone-input`. Maybe we should check if `testId` is specified before setting it?